sematic/api/endpoints/events.py

```diff
@@ -88,6 +130,7 @@ def _call_broadcast_endpoint(
             endpoint=url,
             kwargs=dict(json=json_payload),
             user=user,
+            validate_version_compatibility=False,
```

Comment:
The SocketIO server doesn't serve up the `/meta/versions` endpoint anymore. This call is from the main API server to the SocketIO one. But it's better to not have it anyway; since we're making a call between two servers in the same deployment there's no need to validate compatibility,

Reply:
this might be roughly accurate, but it isn't guaranteed to be true.  if there are a bunch of replicas and the disruption budget is low (or the default of `1`), when you upgrade the server it will bring up one pod at a time and keep any leftover old pods, thereby resulting in mixed versions operating at the same time.  we could just disallow this though.

Reply:
In that case, the original version compatibility check would have failed anyway, and we'd be relying on retries to resolve it. Seems reasonable to me to do the same here--if there is a mismatch that causes a failure in the API call, let retry handle it. Plus this is a bit more robust--the version compatibility check will require that the entire SDK is version compatible, whereas all we really need for this call to succeed is compatibility in the single endpoint we're calling (which changes less often).